Adding -Wdeclaration-after-statement can cause problems for people
trying to build extensions with gcc-3.3 (which doesn't support it).

diff -Naur --exclude=debian perl-5.8.8.orig/Configure perl-5.8.8/Configure
--- perl-5.8.8.orig/Configure	2006-01-09 01:51:03.000000000 +1100
+++ perl-5.8.8/Configure	2006-02-26 13:24:19.000000000 +1100
@@ -4775,19 +4775,19 @@
 			echo "Nope, it doesn't, but that's ok." 2>&1
 		fi
 
-		echo "Checking if your compiler accepts -Wdeclaration-after-statement" 2>&1
-		echo 'int main(void) { return 0; }' > gcctest.c
-		if $cc -Wdeclaration-after-statement -o gcctest gcctest.c; then
-			echo "Yes, it does." 2>&1
-			case "$ccflags" in
-			*-Wdeclaration-after-statement*)
-				echo "Leaving current flags $ccflags alone." 2>&1
-				;;
-			*) dflt="$dflt -Wdeclaration-after-statement" ;;
-			esac
-		else
-			echo "Nope, it doesn't, but that's ok." 2>&1
-		fi
+#		echo "Checking if your compiler accepts -Wdeclaration-after-statement" 2>&1
+#		echo 'int main(void) { return 0; }' > gcctest.c
+#		if $cc -Wdeclaration-after-statement -o gcctest gcctest.c; then
+#			echo "Yes, it does." 2>&1
+#			case "$ccflags" in
+#			*-Wdeclaration-after-statement*)
+#				echo "Leaving current flags $ccflags alone." 2>&1
+#				;;
+#			*) dflt="$dflt -Wdeclaration-after-statement" ;;
+#			esac
+#		else
+#			echo "Nope, it doesn't, but that's ok." 2>&1
+#		fi
 		;;
 	esac
 	;;
